internal/lang/funcs/string.go

```diff
@@ -164,3 +166,69 @@ func Replace(str, substr, replace cty.Value) (cty.Value, error) {
 func StrContains(str, substr cty.Value) (cty.Value, error) {
 	return StrContainsFunc.Call([]cty.Value{str, substr})
 }
+
+// This constant provides a placeholder value for filename indicating
+// that no file is needed for templatestring.
+const (
+	TemplateStringFilename = "NoFileNeeded"
```

Comment:
This constant seems to be internal and should not be imported.

Reply:
noted